config/rewrites.json

```diff
@@ -153,5 +153,148 @@
+            "status": 301
+        },
+        {
+            "description": "Cloud DNS redirect",
+            "from": "^\\/api-ref-dns(\\.html)?",
+            "to": "/docs/cloud-dns/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Files redirect",
+            "from": "^\\/api-ref-files.html$",
+            "to": "/docs/cloud-files/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Images redirect",
+            "from": "^\\/api-ref-images(\\.html)?",
+            "to": "/docs/cloud-images/v2/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Load Balancers redirect",
+            "from": "^\\/api-ref-load-balancers(\\.html)?",
+            "to": "/docs/cloud-load-balancers/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Monitoring redirect",
+            "from": "^\\/api-ref-monitoring(\\.html)?",
+            "to": "/docs/cloud-monitoring/v1/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Networks redirect",
+            "from": "^\\/api-ref-networks(\\.html)?",
+            "to": "/docs/cloud-networks/v2/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Orchestration redirect",
+            "from": "^\\/api-ref-orchestration(\\.html)?",
+            "to": "/docs/cloud-orchestration/v1/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Queues redirect",
+            "from": "^\\/api-ref-queues(\\.html)?",
+            "to": "/docs/cloud-queues/v1/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Rackspace CDN redirect",
+            "from": "^\\/api-ref-raxCDN(\\.html)?",
+            "to": "/docs/cdn/v1/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Email and Apps API redirect",
+            "from": "^\\/api-ref-email-and-apps(\\.html)?",
+            "to": "/docs/",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Identity API redirect",
+            "from": "^\\/api-ref-identity(\\.html)?",
+            "to": "/docs/cloud-identity/v2/developer-guide/#document-api-reference",
+            "rewrite": false,
+            "status": 301
+        }
+        ,
```

Comment:
JSON style nit: would you mind putting the comma on the same line as the `}` above, like the rest?